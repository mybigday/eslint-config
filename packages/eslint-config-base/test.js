const util = require('util')
const exec = util.promisify(require('child_process').exec)

test('Base config', async () => {
  try {
    await exec(
      'eslint --config packages/eslint-config-base/index.js packages/eslint-config-base/fixtures/',
    )
  } catch (e) {
    expect(e.stdout).toMatchSnapshot()
  }
})
