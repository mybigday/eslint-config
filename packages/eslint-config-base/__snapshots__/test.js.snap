// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base config 1`] = `
"
/Users/jhen/workspace/mybigday/eslint-config/packages/eslint-config-base/fixtures/import.js
  2:8   error  'React' is defined but never used          no-unused-vars
  2:17  error  'Component' is defined but never used      no-unused-vars
  2:28  error  'PureComponent' is defined but never used  no-unused-vars
  2:49  error  Unable to resolve path to module 'react'   import/no-unresolved
  4:24  error  Unable to resolve path to module 'react'   import/no-unresolved

/Users/jhen/workspace/mybigday/eslint-config/packages/eslint-config-base/fixtures/unicorn.js
  1:0  error  Specify the rules you want to disable  unicorn/no-abusive-eslint-disable

âœ– 6 problems (6 errors, 0 warnings)

"
`;
